File: programmers/[programmers] 최소직사각형.py
def solution(sizes):
    answer = 0
    max_n = max_m = 0
    for n, m in sizes:
        if n<m: n, m = m, n
        if n>max_n: max_n = n
        if m>max_m: max_m = m
    answer = max_n*max_m
    return answer

File: programmers/[programmers] 최소직사각형.js
function solution(sizes) {
  const answer = [0, 0];
  for (let i = 0; i < sizes.length; i++) {
    const a = Math.max(...sizes[i]);
    const b = Math.min(...sizes[i]);
    answer[0] = Math.max(answer[0], a);
    answer[1] = Math.max(answer[1], b);
  }
  return answer[0] * answer[1];
}
